### DISCUSSION TITLE
Compare and merge not only branches but also tags

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#439] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hi!
right now in the merge of configuration feature it is only possible to do it in respect of branches.
I find this feature also useful to compare differences between the branches. 

I would love if it is possible to also do the comparison between tags. I use case is do a diff between tag actually released in production environment and the next to be released tag.

What you do think?

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
